x-pack/plugins/security/server/deprecations/kibana_user_role.ts

```diff
@@ -115,10 +115,10 @@ async function getUsersDeprecations(
             'xpack.security.deprecations.kibanaUser.usersDeprecationCorrectiveAction',
             {
               defaultMessage:
-                'Remove the "{userRoleName}" role from all users and add the "{adminRoleName}" role. The affected users are: {users}.',
+                'Remove the "{kibanaUserRoleName}" role from all users and add the "{kibanaAdminRoleName}" role. The affected users are: {users}.',
```

Comment:
I changed the string template keys to match the actual role name for clarity and consistency.